test/ng/qSpec.js

```diff
@@ -2112,6 +2125,51 @@ describe('q', function() {
     });
 
 
+
+    describe('when exceptionHandler is called', function() {
```

Comment:
I think this is more relevant to be included in SE source code as we don't normally run angular ng library test source code we just assume the angular library "just works".

Potentially we don't need that many just a simple check that our built in exception handlers will get called by default should be sufficient. If this is coming from the angular PR you're back porting then I'd leave this in.